content/reference/admin/private_locations/configuration/ec2/index.md

```diff
@@ -51,8 +51,8 @@ control-plane {
       security-groups = ["sg-mysecuritygroup"]
       # Instance type
       instance-type = "c5.xlarge"
-      # VPC
-      vpc = "vpc-a"
+      # Spot instances (optional)
+      # spot = true
```

Comment:
Spot instances (optional) => Spot instances (optional, false by default)

so the default behavior is clear